nbstripout.py

```diff
@@ -240,6 +258,10 @@ def main():
                       help='Check if nbstripout is installed in current repository')
     task.add_argument('--status', action='store_true',
                       help='Print status of nbstripout installation in current repository and configuration summary if installed')
+    parser.add_argument('-c', '--count', action='store_true', help="""
+                        Do not strip the execution count/prompt number""")
+    parser.add_argument('-o', '--output', action='store_true', help="""
+                        Do not strip the output""")
```

Comment:
For consistency, could you put the help option on the 2nd line and enclose the help text in `''`